provisioning/service/src/DeviceRegistrationNotFoundException.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Common;
+
+namespace Microsoft.Azure.Devices.Provisioning.Service
+{
+    using System;
+    using System.Runtime.Serialization;
+    using Microsoft.Azure.Devices.Common.Exceptions;
+
+#if !WINDOWS_UWP
+    [Serializable]
```

Comment:
Is this not valid on UWP? Do we have an option to do one that is? something like [this?](https://stackoverflow.com/questions/32446989/system-serializable-attribute-gone-in-windows-10-uwp-apps)